changelog/pricing/api.mdx

```diff
@@ -0,0 +1,14 @@
+---
+sidebar_position: 3
+title: Pricing API
+---
+
+# Changelogs
+
+<Update label="2025-01-15" description="v0.0.6">
+    ### New Features
+    - **Pricing API Marketplace** is now available in early access! Upload custom pricing data for cloud providers not natively supported or leverage significant pricing advantages. The marketplace allows you to create custom pricing entries that integrate seamlessly with our recommendations engine.
+
+    ### Updates
```

Comment:
@tiemma  good place to chip in that file/disk based reads are now gone and if you have any numbers for performance boost. that would be great

Reply:
Oh, that is all invisible to the customer. I don't think there's anything extra